docs/docs/getting-started/observability.md

```diff
@@ -474,123 +474,6 @@ View the Keptn Applications Dashboard and you should see the DORA metrics and an
 
 ![deployment trace](./assets/deploymenttrace.png)
 
-## Step 14: More control over KeptnApp
-
```

Comment:
The part describing pre/post-deployment tasks and evaluations were moved to lifecycle-management guide, as it does not have much to do with observability